js/big-picture.js

```diff
@@ -0,0 +1,67 @@
+import {isEscapeKey} from './util.js';
+
+const bigPicture = document.querySelector('.big-picture');
+const commentList = bigPicture.querySelector('.social__comments');
+const commentListItem = bigPicture.querySelector('.social__comment');
+
+
+const renderComments = (comments) => {
+  commentList.innerHTML = '';
+  comments.forEach((comment) => {
+    const newComment = commentListItem.cloneNode(true);
+    newComment.querySelector('.social__picture').src = comment.avatar;
+    newComment.querySelector('.social__text').textContent = comment.message;
+    commentList.append(newComment);
+  });
+};
+
+const commentsCount = bigPicture.querySelector('.social__comment-count');
+const commentsLoader = bigPicture.querySelector('.comments-loader');
+
+
+const showComments = (comments) => {
+  const count = 5;
+  if(comments.length <= count) {
+    commentsCount.textContent = `${commentList.children.length} из ${commentList.children.length} комментариев`;
+    commentsLoader.classList.add('hidden');
+  }
+};
+
+const renderPictureComments = ({url, description, likes}) => {
+  bigPicture.querySelector('.big-picture__img').querySelector('img').src = url;
```

Comment:
Давай, чтобы не искать элементы при каждом вызове, вынесем их выше в переменные по аналогии с, например, commentList и commentListItem

Reply:
```suggestion
  bigPicture.querySelector('.big-picture__img img').src = url;
```

Чуть сложнее селектор зато искать два раза не придется))

Reply:
И давай еще при будем затирать `.src = ''` закрытии bigPicture 
Сейчас если я сначала открою bigPicture с одним изображением, а затем с другим, то будет сайд эффект - я какое-то время после открытия второго изображения буду видеть перовое

Reply:
fixed [first](https://github.com/htmlacademy-javascript/2262511-kekstagram-28/pull/7#discussion_r1142419875)
fixed [second](https://github.com/htmlacademy-javascript/2262511-kekstagram-28/pull/7#discussion_r1142420337)